lib/rspec/support/spec/library_wide_checks.rb

```diff
@@ -46,12 +46,10 @@ def load_all_files(files, preamble, postamble=nil)
   define_method :load_all_lib_files do
     files = all_lib_files - lib_test_env_files
     preamble  = ['orig_loaded_features = $".dup', preamble_for_lib]
-    postamble = [
-      'loaded_features = ($" - orig_loaded_features).join("\n")',
-      "File.open('#{loaded_features_outfile}', 'w') { |f| f.write(loaded_features) }"
-    ]
+    postamble = ['puts ($" - orig_loaded_features).join("\n")']
 
-    load_all_files(files, preamble, postamble)
+    @loaded_feature_lines, *rest = load_all_files(files, preamble, postamble)
+    [""] + rest
```

Comment:
This matches how it was before, and also your originally suggested replacement...